guides/source/active_record_migrations.md

```diff
@@ -647,31 +797,55 @@ end
 ```
 
 INFO: Tables with composite primary keys require passing array values rather
-than integer IDs to many methods. See also the [Active Record Querying](
-active_record_querying.html) guide to learn more.
+than integer IDs to many methods. See also the [Active Record Composite Primary
+Keys](active_record_composite_primary_keys.html) guide to learn more.
 
-### When Helpers aren't Enough
+### Execute Arbitrary SQL
 
-If the helpers provided by Active Record aren't enough you can use the [`execute`][]
-method to execute arbitrary SQL:
+If the helpers provided by Active Record aren't enough, you can use the
+[`execute`][] method to execute arbitrary SQL. For example,
 
 ```ruby
-Product.connection.execute("UPDATE products SET price = 'free' WHERE 1=1")
+class UpdateProductPrices < ActiveRecord::Migration[8.0]
+  def up
+    execute "UPDATE products SET price = 'free' WHERE 1=1;"
```

Comment:
I think the "1=1" condition is unnecessary here. This example is updating all rows.